ember-aria-utilities/src/modifiers/aria-grid.ts

```diff
@@ -49,24 +49,21 @@ export default gridData;
  ************************************************/
 
 // Grid -> MutationObserver
-const mutationObservers = new WeakMap<HTMLElement, MutationObserver>();
```

Comment:
nice find -- I think, in spirit, this wanted to be a WeakRef (where we run some cleanup code during garbage collection), but we're relying on the Modifier cleanup to handle removing the observer